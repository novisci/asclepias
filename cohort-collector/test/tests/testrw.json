{"example":[{"totalProcessed":7,"attritionInfo":[{"attritionLevel":{"contents":[1,"dummy"],"tag":"ExcludedBy"},"attritionCount":2},{"attritionLevel":{"tag":"Included"},"attritionCount":5}]},{"contents":{"cohortData":[["a",[5,true]],["b",[5,true]],["c",[10,false]],["d",[99,true]],["f",[86,true]]],"attributes":[{"name":"myVar1","type":"Count","attrs":{"getDerivation":"","getShortLabel":"somelabel","getPurpose":{"getRole":["Outcome"],"getTags":[]},"getLongLabel":"another label"}},{"name":"myVar2","type":"Bool","attrs":{"getDerivation":"","getShortLabel":"","getPurpose":{"getRole":[],"getTags":[]},"getLongLabel":""}}]},"tag":"RW"}]}