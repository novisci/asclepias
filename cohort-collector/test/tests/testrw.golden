{"example":[{"attritionInfo":[[{"tag":"SubjectHasNoIndex"},0],[{"contents":[1,"dummy"],"tag":"ExcludedBy"},0],[{"tag":"Included"},6]],"totalUnitsProcessed":6,"totalSubjectsProcessed":6},{"contents":{"cohortData":[[["a",["2010-07-06","2010-07-07"]],[5,true]],[["b",["2010-07-06","2010-07-07"]],[5,true]],[["d",["2010-07-06","2010-07-07"]],[5,true]],[["e",["2010-07-06","2010-07-07"]],[5,true]],[["f",["2010-07-06","2010-07-07"]],[5,true]],[["g",["2010-07-06","2010-07-07"]],[5,true]]],"attributes":[{"name":"myVar1","type":"Count","attrs":{"getShortLabel":"somelabel","getPurpose":{"getRole":["Outcome"],"getTags":[]},"getLongLabel":"another label","getDerivation":""}},{"name":"myVar2","type":"Bool","attrs":{"getShortLabel":"","getPurpose":{"getRole":[],"getTags":[]},"getLongLabel":"","getDerivation":""}}]},"tag":"RW"}]}